bandit/blacklists/calls.py

```diff
@@ -569,4 +569,65 @@ def gen_blacklist():
         'attacks. Consider using tmpfile() instead.'
         ))
 
+    # updated rulesets starts here
+
+    sets.append(utils.build_conf_dict(
```

Comment:
We already have a bandit plugin to check for use of yaml.load. See https://github.com/PyCQA/bandit/blob/master/bandit/plugins/yaml_load.py